library/core/src/slice/mod.rs

```diff
@@ -78,6 +78,15 @@ pub use index::check_range;
 #[lang = "slice"]
 #[cfg(not(test))]
 impl<T> [T] {
+    /// The maximum, inclusive, length such that the slice is shorter than `isize::MAX` bytes.
```

Comment:
> shorter than

The comment says 'shorter than', but the code might also result in 'equal to'. Which is correct?

Reply:
It should be inclusive here as well, instead of shorter. The original comment from `slice::from_raw_parts` says

> must be no larger than isize::MAX

I must have messed this up writing the comment.